open_lm/data.py

```diff
@@ -421,6 +452,7 @@ def get_wds_dataset(
         dataset = RandomMix(datasets, probs=args.train_data_mix_weights, longest=True)
```

Comment:
Can we throw a warning here, saying mixing is happening at training time it is preferred to mix during tokenization?

Reply:
Included this.